### PR TITLE
Add Wienerroither bakery chain

### PR DESCRIPTION
**Pull request by haansn08**
Wienerroither is a bakery chain which can be found in Carinthia/Austria:
https://www.wienerroither.com/filialen

### CODE REVIEW

**Comment by LaoshuBaby** [comment]
Hello, welcome to NSI commnity, thank you for your first PR.

But notice that if this brand doesn't link to a wikidata, it will not be suggested in ideditor.

You can create one and add `brand:wikidata="Q……",`

**Comment by kjonosm** [comment]
Here's the Wikidata item: https://www.wikidata.org/wiki/Q110474658

File: data/brands/shop/bakery.json [hunk 1]
```diff
@@ -1726,6 +1726,15 @@
         "shop": "bakery"
       }
     },
+    {
+      "displayName": "Wienerroither",
+      "locationSet": {"include": ["at"]},
+      "tags": {
+        "brand": "Wienerroither",
```
Comment (LaoshuBaby) [inline data/brands/shop/bakery.json:1733]:
```suggestion
        "brand": "Wienerroither",
        "brand:wikidata": "Q110474658",
```

**Comment by haansn08** [comment]
I'm not sure if the current Wikidata item fulfills the notability criteria. Brands with social media IDs and no links to other items could get deleted.

**Comment by UKChris-osm** [comment]
> I'm not sure if the current Wikidata item fulfils the notability criteria. Brands with social media IDs and no links to other items could get deleted.

It's always a possibility unfortunately, but all Wikidata's have to start somewhere.

**Pull request merged**